enterprise_access/apps/api_client/constants.py

```diff
@@ -8,3 +8,10 @@
     RequestsConnectionError,
     RequestsTimeoutError,
 )
+
+# License status choices
+ACTIVATED_LICENSE = 'activated'
+ASSIGNED_LICENSE = 'assigned'
+UNASSIGNED_LICENSE = 'unassigned'
+REVOKED_LICENSE = 'revoked'
+REVOCABLE_LICENSE_STATUSES = [ACTIVATED_LICENSE, ASSIGNED_LICENSE]
```

Comment:
nit: you could use a class to namespace these constants so that the constant names read a little clearer
```python
class LicenseStatuses:
    ACTIVATED = 'activated'
    ASSIGNED = 'assigned'
    # ... etc.
    REVOCABLE_STATUSES = [ACTIVATED, ASSIGNED]
```
and then you can use those like `LicenseStatuses.ACTIVATED`.